crates/red_knot_python_semantic/resources/mdtest/attributes.md

```diff
@@ -995,15 +1003,15 @@ functions are instances of that class:
 ```py
 def f(): ...
 
-reveal_type(f.__defaults__)  # revealed: @Todo(full tuple[...] support) | None
-reveal_type(f.__kwdefaults__)  # revealed: @Todo(generics) | None
+reveal_type(f.__defaults__)  # revealed: @Todo(full tuple[...] support)
+reveal_type(f.__kwdefaults__)  # revealed: @Todo(generics)
 ```
 
 Some attributes are special-cased, however:
 
 ```py
-reveal_type(f.__get__)  # revealed: @Todo(`__get__` method on functions)
-reveal_type(f.__call__)  # revealed: @Todo(`__call__` method on functions)
+reveal_type(f.__get__)  # revealed: <method-wrapper `__get__` of `f`>
+reveal_type(f.__call__)  # revealed: <bound method `__call__` of `Literal[f]`>
```

Comment:
I didn't introduce any special handling of `__call__` in this PR, but we might want to model this as `<method-wrapper '__call__' of 'f'>`, similar to `__get__`, if we need any special-casing in `Type::Call` for it.